integration-tests/support/pages/tabs/LatestCommitsTabPage.ts

```diff
@@ -12,14 +13,35 @@ export class LatestCommitsTabPage {
   mergePR(owner: string, repoName: string, pullNumber: number, commitTitle: string, commitMessage: string) {
     const body = { commit_title: `${commitTitle}`, commit_message: `${commitMessage}` }
     Common.githubRequest('PUT', `https://api.github.com/repos/${owner}/${repoName}/pulls/${pullNumber}/merge`, body).then((result) => {
-      cy.wrap(JSON.stringify(result.body)).as(commitTitle)
+      expect(result.body.merged).to.be.true
+      Cypress.env(`${commitTitle}_SHA`, result.body.sha)
```

Comment:
Is thins env used somewhere?

Reply:
Yes, its being used to validate commit SHA on commit details page at the end of test case's.

Reply:
Could you please share the link to it? I can't find it. Thx

Reply:
https://github.com/openshift/hac-dev/pull/531/files#diff-34bbd6e516cc0bfe97e53da7ec8f030544685c68a523bf5029bfa6658f69164fR157

Reply:
So you will remove it, right? Same goes for line 27 - 
```
Cypress.env(`${commitMessage}_SHA\`, result.body.commit.sha)` 
```

Reply:
removed previous one. running the test again to check everything fine 😉

Reply:
So you mean
```
${componentInfo.get('commitMessageUpdate')}_SHA`)
```
should equal to 
```
${commitMessage}_SHA
```
It's a little brainfuck to me :laughing:

Reply:
`${commitMessage}_SHA` is in method so just its just `commitMessage` as same method could be reused at some other place and it will save the SHA of that commit to whatever `commitMessage`  as a unique env var.

`commitMessageUpdate` is the updated commit so I kept as it is.

I know its bit confusing please suggest a better solution I can update the test.

I wanted avoid passing the `HashMap` to the method and setting the commit SHA
also wanted to avoid unnecessary API call to get the commit SHA.

I also tried using the .as() but its losing its context on different `describe`

Reply:
Ok, thank you for the explanation. So you do this to name variables dynamically to differentiate between different commits. Two commits may have the same message, but I guess it's up to the tester to keep that in mind. I don't have any better solution now, so let's keep it as it is.

Reply:
Yes, it will be dynamic based on the `commitMessage` assuming every other commit tester mades are unique.

I will update the key Name from `commitMessageUpdate` to `updatedCommitMessage` and will add one line comment explaining from where the value is coming.

Hope this will avoid some confusions 🙂

Reply:
updated